StringSteamTest.cpp

```diff
@@ -0,0 +1,45 @@
+#include "TestHarness.h"
+
+#include <string>
+using namespace std;
+
+TEST(doubleToStream, stringstream)
+{
+    stringstream message; 
+    message << 5.0;
+    double value;
+    message >> value;
+    CHECK_DOUBLES_EQUAL(5.0, value, numeric_limits<double>::epsilon())
```

Comment:
you'd think that numeric_limits::epsilon would be the right thing to use here, but for complicated reasons explained [here](https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/) , you should probably be using a very small value here such as 0.0001 for the tolerance.